system_reports/osx-xcode-11.4.x.log

```diff
@@ -0,0 +1,1261 @@
+ (i) Ignore Errors: enabled
+
+#
+# This System Report was generated by: https://github.com/bitrise-io/osx-box-bootstrap/blob/master/system_report.sh
+#  Pull Requests are welcome!
+#
+
+
+=== Revision / ID ======================
+* BITRISE_OSX_STACK_REV_ID: v2020_01_15_1
+========================================
+
+
+=== Pre-installed tool versions ========
+* Go: go version go1.13.7 darwin/amd64
+* Python: Python 2.7.16
+* Node.js: v10.19.0
+* NPM: 6.13.7
+* Yarn: 1.22.0
+* Java: openjdk version "1.8.0_242"
+OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_242-b08)
+OpenJDK 64-Bit Server VM (AdoptOpenJDK)(build 25.242-b08, mixed mode)
+* Flutter: Flutter 1.12.13+hotfix.5 • channel stable • https://github.com/flutter/flutter.git
+
+* git: git version 2.25.0
+* git lfs: git-lfs/2.10.0 (GitHub; darwin amd64; go 1.13.6)
+* mercurial/hg: Mercurial Distributed SCM (version 5.3)
+* curl: curl 7.64.1 (x86_64-apple-darwin19.0) libcurl/7.64.1 (SecureTransport) LibreSSL/2.8.3 zlib/1.2.11 nghttp2/1.39.2
+* wget: GNU Wget 1.20.3 built on darwin19.0.0.
+* rsync: rsync  version 2.6.9  protocol version 29
+* unzip: UnZip 6.00 of 20 April 2009, by Info-ZIP.  Maintained by C. Spieler.  Send
+* zip: This is Zip 3.0 (July 5th 2008), by Info-ZIP.
+* tar: bsdtar 3.3.2 - libarchive 3.3.2 zlib/1.2.11 liblzma/5.0.5 bz2lib/1.0.6
+* tree: tree v1.8.0 (c) 1996 - 2018 by Steve Baker, Thomas Moore, Francesc Rocher, Florian Sesser, Kyosuke Tokoro 
+
+* brew: Homebrew 2.2.5
+Homebrew/homebrew-core (git revision b394; last commit 2020-02-11)
+Homebrew/homebrew-cask (git revision a929e; last commit 2020-02-11)
+./system_report.sh: line 58: xctool: command not found
```

Comment:
./system_report.sh: line 58: xctool: command not found
Did you open xcode and run a test on a clean project? 🧐

Reply:
oh it shouldn't be installed, and in fact it should be removed from the system_report.sh but otherwise lgtm